quad/hermite.go

```diff
@@ -127,8 +126,9 @@ func (h Hermite) locationsAsy(n int) (x, w []float64) {
 
 func (h Hermite) locationsAsy0(n int) (x, w []float64) {
 	eps := math.Nextafter(1, math.Inf(1)) - 1
+	x0pts := make([]float64, n/2+n%2)
```

Comment:
I don't think so. The difficulty of using it in parallel go way up for not that much benefit.